src/com/mostc/pftt/runner/PhptTestPreparer.java

```diff
@@ -30,7 +30,7 @@ public PreparedPhptTestCase prepare(PhptTestCase test_case, FileSystemScenario f
 		prep.base_file_name = FileSystemScenario.basename(test_case.getBaseName()).replace("+", "");
 		
 		//
-		if (false /* TODO !AzureWebsitesScenario.check(fs) */) {
+		if (true /* TODO !AzureWebsitesScenario.check(fs) */) {
```

Comment:
I have something similar to this but reflects what is commented out. The AzureWebsitesScenario is currently not working, so I had it as `if(fs instanceof AzureKuduVFSScenario) {`. However, I am not sure how important that is. 

Later I will add a separate PR to delete the leftover SKIPIF temp files.

Reply:
@lavturo, `(fs instanceof AzureKuduVFSScenario)` is false when running locally (for me), so I think we should stick with the hard-coded `true` for now. We can still revisit this later.

And yes, please provide a PR for deleting the leftover temp files.